python/lsst/jointcal/jointcal.py

```diff
@@ -172,6 +172,16 @@ class JointcalConfig(pexConfig.Config):
         doc="Write initial/final fit files containing the contributions to chi2.",
         default=False
     )
+    metricOutputFile = pexConfig.Field(
+        dtype=str,
+        doc="Name of the file to write metric measurements to.",
+        default='jointcal_measurements.json'
+    )
+    metricPackageName = pexConfig.Field(
```

Comment:
I don't think this should be a configurable here. Jointcal metrics are defined in `verify_metrics`: why would they ever be defined elsewhere?

Reply:
I'm happy to remove this as a config.  I just thought it would be handy if we envision having metrics elsewhere.  We can always add it in if it's a problem later.

Reply:
Done.